src/script/user/UserRepository.ts

```diff
@@ -186,41 +232,42 @@ export class UserRepository {
             this.logger.log(`Loaded state of '${users.length}' users from database`, users);
 
             const mappingPromises = users.map(user => {
-              return this.get_user_by_id(user.id).then(userEntity => userEntity.availability(user.availability));
+              return this.get_user_by_id(user.id).then(userEntity => userEntity.availability(user.availability()));
```

Comment:
Small fix

Reply:
wow! Maybe we never saw it cause ko resolves the observable automatically... (?)

Reply:
Maybe, but then the knockout types would be wrong.